apps/api/src/app/admin/admin.controller.ts

```diff
@@ -243,13 +247,30 @@ export class AdminController {
       );
     }
 
+    if (dataSource === 'MANUAL' && isDryRun && isToday(date)) {
+      //const marketData = await this.manualService.scrape()
+      Logger.log(`Check ${symbol} via scraperConfiguration`, dataSource);
+    }
+
     return this.dataGatheringService.gatherSymbolForDate({
       dataSource,
       date,
+      isDryRun,
       symbol
     });
   }
 
+  @Post('test-scraper')
```

Comment:
Instead of introducing a new endpoint, can you use the one specified in https://github.com/ghostfolio/ghostfolio/issues/2292.

Reply:
How would I pass the scraper config? I think this is cleaner instead of bloating the other route. But do there exist an easy way to pass the configured scraper config to that route @dtslvr ?

Reply:
Fair enough. I was not aware that the configuration may not have been saved yet.

Reply:
Ok I will clean up code and fix your comment tomorrow and the ping you